---
layout: archive
title: "Publications"
permalink: /publications/
author_profile: true
---

My research operates at the intersection of computational biology, biophysics, and machine learning. I specialize in developing and applying high-throughput methods and predictive models to understand the rules of protein interactions. The goal of my work is to engineer novel biologics and build robust data pipelines that translate complex biological information into actionable insights for drug discovery.
I have authored and co-authored several publications in high-impact journals, including Nature Communications, PLoS Computational Biology, and Human Mutation. My work progressed from foundational biophysics to high-throughput data and finally to predictive modeling.

For an overview of citations of my papers, please view my [Google Scholar profile](https://scholar.google.ca/citations?user=faDYpP8AAAAJ&hl=en).


## High-Throughput Screening & Systems Biology üî¨
This section showcases your ability to generate and analyze large-scale biological data, a crucial skill in modern biology. It bridges the gap between single-molecule studies and large-scale modeling.
  
_________________

‚Ä¢ **Disrupting the Œ±-synuclein-ESCRT interaction with a peptide inhibitor mitigates neurodegeneration in preclinical models of Parkinson‚Äôs disease.** Nature Communications. 2023. _Nim S, O‚ÄôHara DM, Corbi-Verge C, Perez-Riba A, ..., Suneil K Kalia, Lorraine V Kalia, Kim PM._ [[PMID](https://www.ncbi.nlm.nih.gov/pubmed/37076542)] 


‚Ä¢ **The geometric influence on the Cys2His2 zinc finger domain and functional plasticity.** Nucleic Acids Res. 2020. _Mueller AL, Corbi-Verge C*, Giganti DO, Ichikawa DM, Spencer JM, MacRae M, Garton M, Kim PM, Noyes MB._  [[PMID](https://www.ncbi.nlm.nih.gov/pubmed/32383734)], [[CODE](https://gitlab.com/kimlab/zfgeomodes)]

‚Ä¢ **A Multireporter Bacterial 2-Hybrid Assay for the High-Throughput and Dynamic Assay of PDZ Domain-Peptide Interactions.** ACS Synth Biol. 2019. _Ichikawa DM, Corbi-Verge C*, Shen MJ, Snider J, Wong V, Stagljar I, Kim PM, Noyes MB._
[[PMID](https://www.ncbi.nlm.nih.gov/pubmed/30969105)], [[CODE](https://gitlab.com/kimlab/ngskit)]

‚Ä¢ **A PxL motif conveys timely cell cycle substrate dephosphorylation by the Cdc14 phosphatase.** Nat Struct Mol Biol 2018. _Kataria M, Mouilleron S, Moon-Hyeong Seo,  Corbi-Verge C , Kim PM, Uhlmann F._  
 [[PMID](https://www.ncbi.nlm.nih.gov/pubmed/30455435)], [[CODE](https://gitlab.com/kimlab/ngskit)]

‚Ä¢  **Protein engineering by highly parallel screening of computationally designed variants.** Sci Adv. 2016. _Sun MG, Seo MH, Nim S, Corbi-Verge C, Kim PM._ [[PMID](https://www.ncbi.nlm.nih.gov/pubmed/27453948)]

‚Ä¢ **Pooled screening for antiproliferative inhibitors of protein-protein interactions.** Nat Chem Biol. 2016. _Nim S, Jeon J, Corbi-Verge C, Seo MH, Ivarsson Y, Moffat J, Tarasova N, Kim PM._ [[PMID](https://www.ncbi.nlm.nih.gov/pubmed/26900867)]

‚Ä¢ **Strategies to develop inhibitors of motif-mediated protein-protein interactions as drug leads.**  Annu Rev Pharmacol Toxicol. 2017. _Corbi-Verge C*, Garton M, Nim S, Kim PM._ [[PMID](https://www.ncbi.nlm.nih.gov/pubmed/27618737)]

‚Ä¢ **Motif mediated protein-protein interactions as drug targets.** Cell Commun Signal.  2016. _Corbi-Verge C*, Kim PM._ [[PMID](https://www.ncbi.nlm.nih.gov/pubmed/26936767)]



## Predictive Modeling & Machine Learning ü§ñ
This is the culminating section, demonstrating your expertise in using advanced computational techniques to design, predict, and engineer biological systems. This is a highly sought-after skill set.
   
_________________

‚Ä¢ **A Computational Community Blind Challenge on Pan-Coronavirus Drug Discovery Data** 2025. ([Preprint](https://chemrxiv.org/engage/chemrxiv/article-details/6878bef4fc5f0acb52a813f5))

‚Ä¢ **Computational generation of proteins with predetermined three-dimensional shapes using ProteinSolver** STAR protocols 2021. _A Strokach, D Becerra, C Corbi-Verge, A Perez-Riba, PM Kim_ [[PMID](https://www.ncbi.nlm.nih.gov/pubmed/33997819)] 


‚Ä¢ **Fast and flexible protein design using deep graph neural networks** Cell systems 2020. _A Strokach, D Becerra, C Corbi-Verge, A Perez-Riba, PM Kim_ [[PMID](https://www.ncbi.nlm.nih.gov/pubmed/32971019)] 


‚Ä¢ **Predicting changes in protein stability caused by mutation using sequence- and structure-based methods in a CAGI5 blind challenge.** Human Mutation. 2019. _Strokach A, Corbi-Verge C, Teyra J, Kim PM._ [[PMID](https://www.ncbi.nlm.nih.gov/pubmed/31243847)] 
  
‚Ä¢ **Evaluating the predictions of the protein stability change upon single amino acid substitutions for the FXN CAGI5 challenge.** Human Mutation. 2019. _Castrense Savojardo, Maria Petrosino, Giulia Babbi, Samuele Bovo, Carles Corbi‚ÄêVerge, Rita Casadio, Piero Fariselli, Lukas Folkman, Aditi Garg, Mostafa Karimi, Panagiotis Katsonis, Philip M Kim, Olivier Lichtarge, Pier Luigi Martelli, Alessandra Pasquo, Debnath Pal, Yang Shen, Alexey V Strokach, Paola Turina, Yaoqi Zhou, Gaia Andreoletti, Steven E Brenner, Roberta Chiaraluce, Valerio Consalvi, Emidio Capriotti._  [[PMID](https://www.ncbi.nlm.nih.gov/pubmed/31209948)] 
 
‚Ä¢ **Predicting the effect of mutations on protein folding and protein-protein interaction.** Methods Mol Biol. 2019. _Strokach A, Corbi-Verge C, Teyra J, Kim PM._ [[PMID](https://www.ncbi.nlm.nih.gov/pubmed/30298389)]
  
  
‚Ä¢ **Semi-supervised Learning Predicts Approximately One-Third of the Alternative Splicing Isoforms as Functional Proteins.** Cell Rep. 2015. _Hao Y, Colak R, Teyra J, Corbi-Verge C, Ignatchenko A, Hahne H, Wilhelm M, Kuster B, Braun P, Kaida D, Kislinger T, Kim PM._ [[PMID](https://www.ncbi.nlm.nih.gov/pubmed/26146086)], [[CODE](https://gitlab.com/kimlab/pulse)] 


#### Conferences.

‚Ä¢ **Designing novel proteins with high accuracy using deep graph neural networks.**  14th Machine Learning in Computational Biology (MLCB) meeting. _Alexey Strokach, David Becerra, Carles Corbi-Verge, Albert Perez-Riba and Philip M. Kim._ 2019  Vancouver, Canada. [POSTER]

‚Ä¢ **C2H2 Zinc Finger DNA Binding Motif Prediction combining High-throughput experiments and Deep Learning.** 14th Machine Learning in Computational Biology (MLCB) meeting _Carles Corbi-Verge, April Muller, Marcus Noyes and Philip Kim._ 2019. Vancouver, Canada. [POSTER]

‚Ä¢ **De Novo Crystallization Condition Prediction with Deep Learning**. 14th Machine Learning in Computational Biology (MLCB) meeting. _Hyunmin Lee, Zhen Hao Wu, Carles Corbi-Verge, Mac Mok, Sidney Kang, Shun Liao, Zhaolei Zhang and Michael Garton._  2019. Vancouver, Canada. [POSTER]

‚Ä¢ **RNAseq pipeline for the identification and classification of aberrant isoforms on cancer**. Donelly Center for Cellular and Biomolecular Research Retreat. _Corbi-Verge C, Kim PM._ 2017. Ontario, Canada. [POSTER] [[CODE](https://gitlab.com/kimlab/rnapulse)]



## Foundations in Protein Biophysics & Structural Analysis üß¨
This section establishes your core expertise in the fundamental principles governing protein structure, stability, and interactions.

   
_________________

‚Ä¢ **Phage display identification of high-affinity ligands for human TSG101-UEV: A structural and thermodynamic study of PTAP recognition** International Journal of biological macromolecules. 2024. _Javier Murciano-Calles, Alejandro Rodr√≠guez-Mart√≠nez, Andr√©s Palencia, Montserrat And√∫jar-S√°nchez, Manuel Iglesias-Bexiga, Carles Corbi-Verge, Pedro Buzon, Javier Ruiz-Sanz, Jose C Martinez, Horacio Perez-Sanchez, Ana Camara-Artigas, Irene Luque_ [[PMID](https://www.ncbi.nlm.nih.gov/pubmed/38901510)] 

‚Ä¢ **Phage display identification of nanomolar ligands for human NEDD4-WW3: Energetic and dynamic implications for the development of broad-spectrum antivirals** International Journal of biological macromolecules. 2022. _Francisco Castillo, Carles Corbi-Verge, Javier Murciano-Calles, Adela M Candel, Ziying Han, Manuel Iglesias-Bexiga, Javier Ruiz-Sanz, Philip M Kim, Ronald N Harty, Jose C Martinez, Irene Luque_ [[PMID](https://www.ncbi.nlm.nih.gov/pubmed/35257734)] 

‚Ä¢ **Binding Site Plasticity in Viral PPxY Late Domain Recognition by the Third WW Domain of Human NEDD4**  Sci Rep. 2019.  _Iglesias-Bexiga M, Palencia  A, Corbi-Verge C, Martin-Malpartida P, Blanco FJ, Macias MJ, Cobos ES, Luque I._ [[PMID](https://pubmed.ncbi.nlm.nih.gov/31636332)]

‚Ä¢  **Rapid and Accurate Structure‚ÄêBased Therapeutic Peptide Design using GPU Accelerated Thermodynamic Integration.** Proteins 2018. _Garton M,  Corbi-Verge C*, Hu Y, Nim S, Sherborne B,  Kim PM._ [[PMID](https://www.ncbi.nlm.nih.gov/pubmed/30520126)], [[CODE](https://gitlab.com/kimlab/rapid)] 

‚Ä¢ **Allosteric Modulation of Binding Specificity by Alternative Packing of Protein Cores.** J Mol biol 2018. _Ben-David M, Huang H, Sun MGF, Corbi-Verge C, Petsalaki E, Liu K, Gfeller D, Garg P, Tempel W, Sochirca I, Shifman JM, Davidson A, Min J, Kim PM, Sidhu SS._ [[PMID](https://www.ncbi.nlm.nih.gov/pubmed/30471255)]
  
‚Ä¢ **Post-Translational modifications modulate ligand recognition by the third PDZ domain of the MAGUK protein PSD-95.** PLoS One. 2014. _Murciano-Calles J, Corbi-Verge C*, Candel AM, Luque I, Martinez JC._ [[PMID](https://www.ncbi.nlm.nih.gov/pubmed/24587199)]

‚Ä¢ **Two-state dynamics of the SH3-SH2 tandem of Abl kinase and the allosteric role of the N-cap.** Proc Natl Acad Sci U S A. 2013. _Corbi-Verge C*, Marinelli F, Zafra-Ruano A, Ruiz-Sanz J, Luque I, Faraldo-G√≥mez JD._ [[PMID](https://www.ncbi.nlm.nih.gov/pubmed/23959873)]

‚Ä¢ **The role of residue stability in transient protein-protein interactions involved in enzymatic phosphate hydrolysis. A computational study.** Proteins. 2006.  _Bonet J, Caltabiano G, Khan AK, Johnston MA, Corb√≠ C, G√≥mez A, Rovira X, Teyra J, Vill√†-Freixa J._ [[PMID](https://www.ncbi.nlm.nih.gov/pubmed/16374872)]

#### Conferences.

‚Ä¢ **Conformational Distribution of the SH3-SH2 Tandem of the ABL kinase and its Modulation by Intramolecular interactions**.  XXXVII Congreso de la Sociedad Espa√±ola de Bioqu√≠mica y Biolog√≠a Molecular. _Corbi-Verge C, Marinelli F, Zafra-Ruano A, Ruiz-Sanz J, Luque I, Faraldo-G√≥mez JD._ 2014 Granada, Spain. [POSTER]

‚Ä¢ **Long Range Cooperative effects in SH3 Domains From The Src Family.** VIII European Symposium of the Protein Society. _Corbi-Verge C, Martin-Garcia JM, Martinez, JC, Sanz JR, Luque I._ 2009 Zurich, Switzerland. [POSTER]

‚Ä¢ **Specificity on SH3 interactions: long-range effects on the binding energetics of proline-rich ligands to SH3 domains from the SRC family.** 2nd International Conference on molecular Perspectives on Protein-protein interactions. _Corbi-Verge C, Martin-Garcia JM, Martinez, JC, Sanz JR, Luque I._  2008. Dubrovnik, Croatia. [POSTER]
  






