---
layout: archive
title: "Publications"
permalink: /publications/
author_profile: true
---

{% if author.googlescholar %}
  You can also find my articles on <u><a href="{{author.googlescholar}}">my Google Scholar profile</a>.</u>
{% endif %}

{% include base_path %}

{% for post in site.publications reversed %}
  {% include archive-single.html %}
{% endfor %}


### Peptide library design and screening analysis. 
  **Skills:** _Data integration and Visualization, Molecular Biology, Math and Statistics_
  
  **Code:** [ngskit](https://github.com/ccorbi/ngskit),

•    Ichikawa DM, Corbi-Verge C*, Shen MJ, Snider J, Wong V, Stagljar I, Kim PM, Noyes MB. A Multireporter Bacterial 2-Hybrid Assay for the High-Throughput and Dynamic Assay of PDZ Domain-Peptide Interactions. ACS Synth Biol. 2019. PMID: 30969105

•    Kataria M, Mouilleron S, Moon-Hyeong Seo,  Corbi-Verge C , Kim PM, Uhlmann F. A PxL motif conveys timely cell cycle substrate dephosphorylation by the Cdc14 phosphatase. Nat Struct Mol Biol 2018. PMID: 30455435

•    Sun MG, Seo MH, Nim S, Corbi-Verge C, Kim PM. Protein engineering by highly parallel screening of computationally designed variants. Sci Adv. 2016. PMID: 27453948

•    Nim S, Jeon J, Corbi-Verge C, Seo MH, Ivarsson Y, Moffat J, Tarasova N, Kim PM. Pooled screening for antiproliferative inhibitors of protein-protein interactions. Nat Chem Biol. 2016.  PMID: 26900867

•    Corbi-Verge C*, Garton M, Nim S, Kim PM. Strategies to develop inhibitors of motif-mediated protein-protein interactions as drug leads.  Annu Rev Pharmacol Toxicol. 2017. PMID:27618737

•    Corbi-Verge C*, Kim PM. Motif mediated protein-protein interactions as drug targets. Cell Commun Signal.  2016. PMID: 26936767


### Machine learning. 
   **Skills:** _Data engineering and Science, Supervised and Unsupervised models._
   
   **Code:** [PlayGround](https://github.com/ccorbi/PlayGround), [rnapulse](https://gitlab.com/kimlab/rnapulse)
 
  
•    Strokach A, Corbi-Verge C, Teyra J, Kim PM. Predicting the effect of mutations on protein folding and protein-protein interaction. Methods Mol Biol. 2019 PMID: 30298389

•    Hao Y, Colak R, Teyra J, Corbi-Verge C, Ignatchenko A, Hahne H, Wilhelm M, Kuster B, Braun P, Kaida D, Kislinger T, Kim PM. Semi-supervised Learning Predicts Approximately One-Third of the Alternative Splicing Isoforms as Functional Proteins. Cell Rep. 2015. PMID: 26146086

  There is also a follow-up paper of the isoforms history, that unfortunately, we have not published, but you can find the code [here](https://gitlab.com/kimlab/rnapulse).


### Structural computational biology
   **Skills:** _Structural Biology, Molecular Biology, Data analytics, Biophysics, Math and Statistics_
   
   **Code:** [rapid](https://gitlab.com/kimlab/rapid)


•    Garton M,  Corbi-Verge C*, Hu Y, Nim S, Sherborne B,  Kim PM. GPU accelerated free energy calculations allow rapid and highly accurate design of inhibitory peptides. Proteins 2018 PMID: 30520126

•    Ben-David M, Huang H, Sun MGF, Corbi-Verge C, Petsalaki E, Liu K, Gfeller D, Garg P, Tempel W, Sochirca I, Shifman JM, Davidson A, Min J, Kim PM, Sidhu SS. Allosteric Modulation of Binding Specificity by Alternative Packing of Protein Cores. J Mol biol 2018. PMID: 30471255

•    Murciano-Calles J, Corbi-Verge C*, Candel AM, Luque I, Martinez JC. Post-Translational modifications modulate ligand recognition by the third PDZ domain of the MAGUK protein PSD-95. PLoS One. 2014. PMID: 24587199

•    Corbi-Verge C*, Marinelli F, Zafra-Ruano A, Ruiz-Sanz J, Luque I, Faraldo-Gómez JD. Two-state dynamics of the SH3-SH2 tandem of Abl kinase and the allosteric role of the N-cap. Proc Natl Acad Sci U S A. 2013. PMID: 23959873

•    Bonet J, Caltabiano G, Khan AK, Johnston MA, Corbí C, Gómez A, Rovira X, Teyra J, Villà-Freixa J. The role of residue stability in transient protein-protein interactions involved in enzymatic phosphate hydrolysis. A computational study. Proteins. 2006 PMID:16374872
