---
layout: archive
title: "Publications"
permalink: /publications/
author_profile: true
---

For an overview of citations of my papers, please view my [Google Scholar profile](https://scholar.google.ca/citations?user=faDYpP8AAAAJ&hl=en).


## High-Troughput experiments analysis. 
  **Keywords:** _Data integration, Data Visualization, Molecular Biology, Peptides, Protein-Protein Interactions, Systems Biology, Math and Statistics, Supervised models, Computational Biology_
_________________

• **The geometric influence on the Cys2His2 zinc finger domain and functional plasticity.** Nucleic Acids Res. 2020. Mueller AL, Corbi-Verge C*, Giganti DO, Ichikawa DM, Spencer JM, MacRae M, Garton M, Kim PM, Noyes MB.  [[PMID](https://www.ncbi.nlm.nih.gov/pubmed/32383734)], [[CODE](https://gitlab.com/kimlab/zfgeomodes)]

• **A Multireporter Bacterial 2-Hybrid Assay for the High-Throughput and Dynamic Assay of PDZ Domain-Peptide Interactions.** ACS Synth Biol. 2019. Ichikawa DM, Corbi-Verge C*, Shen MJ, Snider J, Wong V, Stagljar I, Kim PM, Noyes MB. 
[[PMID](https://www.ncbi.nlm.nih.gov/pubmed/30969105)], [[CODE](https://gitlab.com/kimlab/ngskit)]

• **A PxL motif conveys timely cell cycle substrate dephosphorylation by the Cdc14 phosphatase.** Nat Struct Mol Biol 2018. Kataria M, Mouilleron S, Moon-Hyeong Seo,  Corbi-Verge C , Kim PM, Uhlmann F.  
 [[PMID](https://www.ncbi.nlm.nih.gov/pubmed/30455435)], [[CODE](https://gitlab.com/kimlab/ngskit)]

•  **Protein engineering by highly parallel screening of computationally designed variants.** Sci Adv. 2016. Sun MG, Seo MH, Nim S, Corbi-Verge C, Kim PM. [[PMID](https://www.ncbi.nlm.nih.gov/pubmed/27453948)]

• **Pooled screening for antiproliferative inhibitors of protein-protein interactions.** Nat Chem Biol. 2016. Nim S, Jeon J, Corbi-Verge C, Seo MH, Ivarsson Y, Moffat J, Tarasova N, Kim PM. [[PMID](https://www.ncbi.nlm.nih.gov/pubmed/26900867)]

• **Strategies to develop inhibitors of motif-mediated protein-protein interactions as drug leads.**  Annu Rev Pharmacol Toxicol. 2017. Corbi-Verge C*, Garton M, Nim S, Kim PM. [[PMID](https://www.ncbi.nlm.nih.gov/pubmed/27618737)]

• **Motif mediated protein-protein interactions as drug targets.** Cell Commun Signal.  2016. Corbi-Verge C*, Kim PM. [[PMID](https://www.ncbi.nlm.nih.gov/pubmed/26936767)]



## Building and design of predictive models.
   **Keywords:** _Data engineering, Structural Biology, Proteins, Protein-Protein Interactions, Data Science, Supervised and Unsupervised models, Structural Computational Biology, Math and Statistics._
_________________

• **Predicting changes in protein stability caused by mutation using sequence- and structure-based methods in a CAGI5 blind challenge.** Human Mutation. 2019. Strokach A, Corbi-Verge C, Teyra J, Kim PM.
  [[PMID](https://www.ncbi.nlm.nih.gov/pubmed/31243847)] 
  
• **Evaluating the predictions of the protein stability change upon single amino acid substitutions for the FXN CAGI5 challenge.** Human Mutation. 2019. Castrense Savojardo, Maria Petrosino, Giulia Babbi, Samuele Bovo, Carles Corbi‐Verge, Rita Casadio, Piero Fariselli, Lukas Folkman, Aditi Garg, Mostafa Karimi, Panagiotis Katsonis, Philip M Kim, Olivier Lichtarge, Pier Luigi Martelli, Alessandra Pasquo, Debnath Pal, Yang Shen, Alexey V Strokach, Paola Turina, Yaoqi Zhou, Gaia Andreoletti, Steven E Brenner, Roberta Chiaraluce, Valerio Consalvi, Emidio Capriotti.  [[PMID](https://www.ncbi.nlm.nih.gov/pubmed/31209948)] 
 
• **Predicting the effect of mutations on protein folding and protein-protein interaction.** Methods Mol Biol. 2019. Strokach A, Corbi-Verge C, Teyra J, Kim PM. [[PMID](https://www.ncbi.nlm.nih.gov/pubmed/30298389)]
  
•  **Rapid and Accurate Structure‐Based Therapeutic Peptide Design using GPU Accelerated Thermodynamic Integration.** Proteins 2018. Garton M,  Corbi-Verge C*, Hu Y, Nim S, Sherborne B,  Kim PM. [[PMID](https://www.ncbi.nlm.nih.gov/pubmed/30520126)], [[CODE](https://gitlab.com/kimlab/rapid)] 
  
• **Semi-supervised Learning Predicts Approximately One-Third of the Alternative Splicing Isoforms as Functional Proteins.** Cell Rep. 2015. Hao Y, Colak R, Teyra J, Corbi-Verge C, Ignatchenko A, Hahne H, Wilhelm M, Kuster B, Braun P, Kaida D, Kislinger T, Kim PM. [[PMID](https://www.ncbi.nlm.nih.gov/pubmed/26146086)], [[CODE](https://gitlab.com/kimlab/pulse)] 

• **RNApulse, A pipeline for identification of novel isoforms.** (In preparation) Corbi-Verge C, Kim PM. [[CODE](https://gitlab.com/kimlab/rnapulse)]



## Protein structure and conformation analysis.
   **Keywords:** _Structural Biology, Proteins, Protein-Protein Interactions, Molecular Biology, Data analytics, Biophysics, Structural Computational Biology, Math and Statistics._
_________________

• **Binding Site Plasticity in Viral PPxY Late Domain Recognition by the Third WW Domain of Human NEDD4**  Sci Rep. 2019.  Iglesias-Bexiga M, Palencia  A, Corbi-Verge C, Martin-Malpartida P, Blanco FJ, Macias MJ, Cobos ES, Luque I. [[PMID](https://pubmed.ncbi.nlm.nih.gov/31636332)]

• **Allosteric Modulation of Binding Specificity by Alternative Packing of Protein Cores.** J Mol biol 2018. Ben-David M, Huang H, Sun MGF, Corbi-Verge C, Petsalaki E, Liu K, Gfeller D, Garg P, Tempel W, Sochirca I, Shifman JM, Davidson A, Min J, Kim PM, Sidhu SS. [[PMID](https://www.ncbi.nlm.nih.gov/pubmed/30471255)]
  
• **Post-Translational modifications modulate ligand recognition by the third PDZ domain of the MAGUK protein PSD-95.** PLoS One. 2014. Murciano-Calles J, Corbi-Verge C*, Candel AM, Luque I, Martinez JC. [[PMID](https://www.ncbi.nlm.nih.gov/pubmed/24587199)]

• **Two-state dynamics of the SH3-SH2 tandem of Abl kinase and the allosteric role of the N-cap.** Proc Natl Acad Sci U S A. 2013. Corbi-Verge C*, Marinelli F, Zafra-Ruano A, Ruiz-Sanz J, Luque I, Faraldo-Gómez JD. [[PMID](https://www.ncbi.nlm.nih.gov/pubmed/23959873)]

• **The role of residue stability in transient protein-protein interactions involved in enzymatic phosphate hydrolysis. A computational study.** Proteins. 2006.  Bonet J, Caltabiano G, Khan AK, Johnston MA, Corbí C, Gómez A, Rovira X, Teyra J, Villà-Freixa J. [[PMID](https://www.ncbi.nlm.nih.gov/pubmed/16374872)]
  
  
## Conferences.

• **Designing novel proteins with high accuracy using deep graph neural networks.**  14th Machine Learning in Computational Biology (MLCB) meeting. Alexey Strokach, David Becerra, Carles Corbi-Verge, Albert Perez-Riba and Philip M. Kim. 2019  Vancouver, Canada. [POSTER]

• **C2H2 Zinc Finger DNA Binding Motif Prediction combining High-throughput experiments and Deep Learning.** 14th Machine Learning in Computational Biology (MLCB) meeting Carles Corbi-Verge, April Muller, Marcus Noyes and Philip Kim. 2019. Vancouver, Canada. [POSTER]

• **De Novo Crystallization Condition Prediction with Deep Learning**. 14th Machine Learning in Computational Biology (MLCB) meeting. Hyunmin Lee, Zhen Hao Wu, Carles Corbi-Verge, Mac Mok, Sidney Kang, Shun Liao, Zhaolei Zhang and Michael Garton.  2019. Vancouver, Canada. [POSTER]

• **RNAseq pipeline for the identification and classification of aberrant isoforms on cancer**. Donelly Center for Cellular and Biomolecular Research Retreat Corbi-Verge C, Kim PM. 2017. Ontario, Canada. [POSTER]

• **Conformational Distribution of the SH3-SH2 Tandem of the ABL kinase and its Modulation by Intramolecular interactions**.  XXXVII Congreso de la Sociedad Española de Bioquímica y Biología Molecular.  Corbi-Verge C, Marinelli F, Zafra-Ruano A, Ruiz-Sanz J, Luque I, Faraldo-Gómez JD. 2014 Granada, Spain. [POSTER]

• **Long Range Cooperative effects in SH3 Domains From The Src Family.** VIII European Symposium of the Protein Society Corbi-Verge C, Martin-Garcia JM, Martinez, JC, Sanz JR, Luque I.  2009 Zurich, Switzerland. [POSTER]

• **Specificity on SH3 interactions: long-range effects on the binding energetics of proline-rich ligands to SH3 domains from the SRC family.** 2nd International Conference on molecular Perspectives on Protein-protein interactions. Corbi-Verge C, Martin-Garcia JM, Martinez, JC, Sanz JR, Luque I.  2008. Dubrovnik, Croatia. [POSTER]

